README.md

```diff
@@ -233,12 +251,31 @@ If you run this command without a `--target-height` it will sync all blocks whic
 available in the pool. KSYNC will automatically exit once a target height is reached, or you can simply exit the sync 
 process by killing KSYNC with CMD+C.
 
+### AUTO-SYNC
+
+Due to the [100MB limitation](#limitations), P2P-Sync was implemented to support KSYNC for any type of Tendermint blockchain.
+However, in comparison to DB-Sync it has some disadvantages, which is why it's recommended to use P2P-Sync only as long as the first block was synced successfully before switching back to DB-SYNC.
+On top of that, you need to run the node to be synced and KSYNC in two different terminals, which isn't that comfortable for developers.
+AUTO-SYNC consists of a process manager, running P2P-Sync or DB-Sync on the one hand and the node to be synced on the other hand.
+Based on the requirements, the process manager will manage which syncing process is required, thereby enabling the best way of syncing the node with the validated blocks.
+If the node is completely synced with the corresponding KYVE pool, it will start normally to find peers through the provided seeds.
+This resolves in the ability to sync a node completely with KYVE data requiring just one command.
+
+#### AUTO-SYNC requirements
+The requirements are similar to the [DB-SYNC requirements](#db-requirements). Everything else will be set up automatically.
+
+#### Sync node with AUTO-SYNC
+To start the syncing process, simply run
+````bash
+ksync start --mode="auto" --home="/Users/<user>/.<chain>" --daemon-path="/Users/<user>/<daemon>" --pool-id=<pool> --rest=<network-api-endpoint> --seeds <p2p.seeds>
```

Comment:
--mode="auto" should be default, therefor it should not be required to add this flag